File: Unity Pepijn/Melee/Assets/enemyLogic.js
#pragma strict

var Health = 100;

function Update ()
{
	if(Health <= 0)
	{
		Dead();
	}
}

function ApplyDammage (TheDammage : int)
{
	Health = Health - TheDammage;
}
function Dead ()
{
	Destroy (gameObject);
}

File: Unity Pepijn/Melee/Assets/enemyLogic.cs
using UnityEngine;
using System.Collections;

public class enemyLogic : MonoBehaviour {
	

	int Health = 100;

	void  Update (){
		if(Health <= 0)
		{
			Dead();
		}
	}
	
	void  ApplyDamage ( int TheDamage  ){
		Debug.Log("in ApplyDamage");
		Health = Health - TheDamage;
	}
	void  Dead (){
		Destroy (gameObject);
	}
}